src/Models/ProjectModel.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+require_once 'src/Entities/ProjectEntity.php';
+
+class ProjectModel
+{
+
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+
+    }
+//Magic comment to ensure the array has to go through project Entity so you can't have a key that isn't in Project Entity
+    /**
+     * @return ProjectEntity []
+     *
+     */
+
+    public function getProjects(): array
```

Comment:
Suggest renaming this to something like `getAll` - the fact that it's in the ProjectModel tells me that it's dealing with projects

Reply:
✅